packages/react/src/themes/teams-dark/components/Checkbox/checkboxVariables.ts

```diff
@@ -0,0 +1,5 @@
+import { CheckboxVariables } from '../../../teams/components/Checkbox/checkboxVariables'
+
+export default (siteVars: any): Partial<CheckboxVariables> => {
+  return {}
```

Comment:
Is this file needed since it's empty?

Reply:
If a tree falls in the forest, and no one is there to hear it, does it make a sound?